pkg/operator/controllers/monitoring/monitoring_controller.go

```diff
@@ -50,23 +50,25 @@ type Config struct {
 	} `json:"alertmanagerMain,omitempty"`
 }
 
-type Reconciler struct {
+type MonitoringReconciler struct {
```

Comment:
All the other reconciler types have a more specific name than "Reconciler"

Reply:
We should probably standartise the other way around to avoid repetion: `monitoring.MonitoringReconciler` vs `monitoring.Reconciler`.

Reply:
Sure, sounds good. I went with the path of least resistance (a single change vs. changing all the rest), but I agree it is nicer to avoid repetition.

Reply:
I have switched it back to Reconciler. I'm leaving this comment unresolved to serve as context in another discussion below.